rpc/api/api.proto

```diff
@@ -25,6 +25,7 @@ message SubmitRequest {
 
 message SubmitResponse {
     int32 roundId = 1;
+    bytes poetId = 2; // TODO: remove this once we have persistent PoET identities that the node knows in advance
```

Comment:
I found the name a bit confusing, because "poet" refers to a proof. How about `serviceId`?
I know it's already implemented in other places, so you can leave it as is if you prefer.